github.js

```diff
@@ -74,14 +76,35 @@ function upsertNavigationInBody(newNavigation, body) {
   }
 }
 
+function checkAndReportInvalidBaseError(e, base) {
+  const { field, code } = get(e, 'body.errors[0]', {});
+  if (field === 'base' && code === 'invalid') {
+    console.log([
+      emoji.get('no_entry'),
+      `\n${emoji.get('confounded')} This is embarrassing. `,
+      `The base branch of ${base.bold} doesn't seem to exist on the remote.`,
+      `\nDid you forget to ${emoji.get('arrow_up')}  push it?`,
+    ].join(''));
+    return true;
+  }
+  return false;
+}
```

Comment:
This caters for scenarios where you are trying to specify a base branch that hasn't been pushed to the remote yet. The intention is to try to help out scatter-brained developers if this scenario does arise.

It certainly did for me, so I added this helpful suggestion to shortcut fixing the problem if anyone else encounters it in the future.

I'm happy to reword it if you feel the tone is inconsistent.